package.json

```diff
@@ -14,9 +14,9 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
+    "@agoric/casting": "^0.4.3-u13.0",
```

Comment:
I figure we should also change "ses" below from `"^1.0.1"` to `"^1.3.0"`

Reply:
Good call, updated